content/en/apps/guides/data/analytics/couch2pg-to-cht-sync-migration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Migrating from couch2pg to CHT Sync"
+weight: 1
+linkTitle: "Couch2pg to CHT Sync Migration"
+description: >
+  Instructions on migrating from couch2pg to CHT Sync for data synchronization and analytics.
+relatedContent: >
+  core/overview/cht-sync
+  core/overview/data-flows-for-analytics/
+---
+
+This page outlines guidelines for migrating from [couch2pg](https://github.com/medic/cht-couch2pg) to the data pipeline based on CHT Sync and CHT Pipeline. One of the main changes in this flow is separating the syncing process from the data transformation, with CHT Pipeline and DBT now handling the latter. This migration requires DBT models with CHT Pipeline instead of SQL views and tables. One thing to note is that the schema for CHT Sync differs from couch2pg, so DBT models will not directly replace the SQL views and tables. For instructions on how to get started with DBT models, refer to the [DBT models guide](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})).
```

Comment:
```suggestion
This page outlines guidelines for migrating from [couch2pg](https://github.com/medic/cht-couch2pg) to the data pipeline based on CHT Sync and CHT Pipeline. One of the main changes in this flow is separating the syncing process from the data transformation, with CHT Pipeline and dbt now handling the latter. This migration requires dbt models with CHT Pipeline instead of SQL views and tables. One thing to note is that the schema for CHT Sync differs from couch2pg, so dbt models will not directly replace the SQL views and tables. For instructions on how to get started with dbt models, refer to the [dbt models guide](({{< relref "apps/guides/data/analytics/building-dbt-models" >}})).
```